README.md

```diff
@@ -593,9 +600,20 @@ export default async function BlogIndex() {
         <h1 className="text-4xl font-bold mb-2">{index.title}</h1>
         <div dangerouslySetInnerHTML={{ __html: index.intro }}></div>
       </div>
-
-      // The rest stays the same
-      // ...
+      {/* The rest is the same as the previous example */}
+      <ul>
```

Comment:
Added this back so people can copy paste the whole thing instead of having to select the updated bits